lib/internal/modules/cjs/loader.js

```diff
@@ -704,14 +705,10 @@ const CircularRequirePrototypeWarningProxy = new Proxy({}, {
   }
 });
 
-// Object.prototype and ObjectPrototype refer to our 'primordials' versions
-// and are not identical to the versions on the global object.
```

Comment:
This is not the case anymore. `primordials.Object` is the same as `global.Object` and `primordials.ObjectPrototype` is the same as `global.Object.prototype`